backend/app/controllers/registration_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class RegistrationController < ApplicationController
+  def signup
```

Comment:
`nit`

Let's adhere to REST. You can rename this as `create`

Then the end point should be `POST /api/registration`

This is mostly a nit